### PR TITLE
ODF template update

### PR DESCRIPTION
**Pull request by gayathrimenath-ibm**

### CODE REVIEW

**Review by akgunjal — APPROVED** [review]
lgtm

File: config-templates/redhat/odf-local/4.7/deployment.yaml [hunk 1]
```diff
@@ -324,8 +324,8 @@ items:
               allowPrivilegeEscalation: false
             resources:
               limits:
-                cpu: 100m
-                memory: 200Mi
+                cpu: 500m
+                memory: 500Mi
```
Comment (mssachan) [inline config-templates/redhat/odf-local/4.7/deployment.yaml:328]: Why are we not updating image for 4.7?

**Review by mssachan — CHANGES_REQUESTED** [review]

File: config-templates/redhat/odf-local/changelog.json [hunk 2]
```diff
@@ -3,43 +3,43 @@
         "4.11": {
             "status": "supported",
             "latest-revision": {
+                "revision": "3",
+                "release-date": "06 March 2023",
+                "cves": "CVE-2022-4415,CVE-2020-10735,CVE-2021-28861, CVE-2022-45061, CVE-2022-40897",
+                "golang": "1.19.6",
+                "ubi": "8.7-1085",
+                "new-features": [],
+                "fixes": []
```
Comment (mssachan) [inline config-templates/redhat/odf-local/changelog.json]:
```
"fixes": [
     "Default values for CPU limit & Memory limit are updated to 500m & 500Mi respectively."
]
```

File: config-templates/redhat/odf-local/4.7/deployment.yaml (hunk 1 above)
Comment (gayathrimenath-ibm) [reply config-templates/redhat/odf-local/4.7/deployment.yaml:328]: 4.7 is deprecated and secure PR has raised and will push the changes to prod

File: config-templates/redhat/odf-local/changelog.json (hunk 2 above)
Comment (gayathrimenath-ibm) [reply config-templates/redhat/odf-local/changelog.json]: sure, I will add

**Review by mssachan — APPROVED** [review]
LGTM

**Pull request merged**